env.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
Unnecessary. This file isn't being executed but evaluated. It uses the currently active shell, so this line does nothing.

Reply:
Same as before, this was for github actions